src/app/Layout/PresentationSettings/PresentationSelectionButtonContainer.tsx

```diff
@@ -2,9 +2,9 @@
 import { connect } from "react-redux";
 import { changePresentation } from "./PresentationActions";
 
-const mapDispatchToProps = (dispatch: any, ownProps: PresentationSelectionButtonProps | undefined) => ({
+const mapDispatchToProps = (dispatch: any, ownProps: PresentationSelectionButtonProps) => ({
     onPress: () => {
-        const props = ownProps as PresentationSelectionButtonProps;
+        const props: PresentationSelectionButtonProps = ownProps;
```

Comment:
This seems superfluous as the type can be inferred from the argument.

Reply:
Huh, must have missed that one.